Source/Blazorise.AntDesign/wwwroot/blazorise.antdesign.css

```diff
@@ -2710,6 +2710,11 @@ li.ant-list-item-actionable {
 .ant-table.ant-table-hover tbody > tr.ant-table-row:hover > td {
   background: #f5f5f5;
 }
+.ant-table tbody tr.ant-table-group {
+  cursor: pointer;
+  background-color: var(--b-theme-light, #f5f5f5);
```

Comment:
hmm do we need this to change ever? Should it always be b-theme-light? are there any configuration the user can do that conflicts with this?

Reply:
The `b-theme-light` is only available when ThemeProvider is used, and Theme is enabled. So yes, we need it.